packages/openapi3/test/tsp-openapi3/parameters.test.ts

```diff
@@ -581,9 +575,7 @@ describe("query", () => {
       expect(Foo.properties.size).toBe(1);
       const fooProperty = Foo.properties.get("foo");
       assert(fooProperty, "foo property not found");
-      expectDecorators(fooProperty.decorators, [
-        { name: "query", args: [{ explode: true, format }] },
-      ]);
+      expectDecorators(fooProperty.decorators, []);
```

Comment:
@chrisradek, is empty the correct expectation here?

Reply:
I don't think that is correct the conversion should be something like 
```
@query @encode(ArrayEncoding.spaceDelimited)
```
and `pipeDelimited` for the other one

Reply:
It is not supported yet by the converter yet as it doesn't create any decorator. I can comment it out with a // TODO: for later. Thoughts?

Reply:
yeah, add `.skip` to the test and file an issue then

Reply:
https://github.com/microsoft/typespec/issues/6351